src/convertToObject.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesInObject = {};
+
+  sourceString
+    .split(';')
+    .map(item => item.trim())
+    .filter(item => item.length > 0)
+    .forEach(item => {
+      const [key, value] = item.split(':').map(v => v.trim());
+
+      stylesInObject[key] = value;
+    });
```

Comment:
that's the perfect usecase for reduce, then you can omit stylesInObject object 😄